Python/Product/Analyzer/Interpreter/LegacyDB/CPythonInterpreter.cs

```diff
@@ -98,6 +97,8 @@ class CPythonInterpreter : IPythonInterpreter {
             return fromDb;
         }
 
+        public Task<IPythonModule> ImportModuleAsync(string name, CancellationToken token) => Task.FromResult(ImportModule(name));
```

Comment:
Unused (since the interface is not added here, and I'm quite happy to not have the interface on this one)